build.gradle

```diff
@@ -1,21 +1,33 @@
-plugins {
-    id 'java'
-    id 'org.jetbrains.intellij' version '0.3.5'
-}
 
 group 'io.vivy'
-version '1.0-SNAPSHOT'
+version '1.0'
 
-sourceCompatibility = 1.8
+allprojects {
 
-repositories {
-    mavenCentral()
-}
+    repositories {
+        mavenCentral()
+    }
 
-dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
 }
 
-intellij {
-    version '2018.2.1'
-}
+
+subprojects {
+    apply plugin: 'java'
+
+    dependencies {
+
+        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.2.0'
+        testCompileOnly 'junit:junit:4.12'
+        testImplementation 'org.junit.jupiter:junit-jupiter-params:5.2.0'
+
+        testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.2.0'
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.2.0'
+
+        compile 'com.puppycrawl.tools:checkstyle:8.12'
```

Comment:
this is not needed by all submodules, only by checkstyle-custom

Reply:
corrected